lib/src/datePicker/date-picker.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input, forwardRef, ViewEncapsulation } from "@angular/core";
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from "@angular/forms";
+import { NgbDateStruct } from "@ng-bootstrap/ng-bootstrap";
+
+export const CUSTOM_INPUT_CONTROL_VALUE_ACCESSOR: any = {
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: forwardRef(() => DatePickerComponent),
+    multi: true,
+};
+
+@Component({
+    selector: "sebng-date-picker",
+    templateUrl: "./date-picker.component.html",
+    styleUrls: ["./date-picker.component.scss"],
+    providers: [CUSTOM_INPUT_CONTROL_VALUE_ACCESSOR],
+    encapsulation: ViewEncapsulation.None,
+})
+export class DatePickerComponent implements ControlValueAccessor {
+    @Input() placeholder: string = "yyyy-mm-dd";
```

Comment:
is there a way to set the datepicker to only show months and years? Like month picker, year picker

Reply:
looking into it. Also considering native month and date picker